### PR TITLE
add php7 xhprof formulas

### PR DESCRIPTION
**Pull request by KonstantinKuklin**
Add formulas:
* php70-xhprof
* php71-xhprof
* php72-xhprof

### CODE REVIEW

**Comment by KonstantinKuklin** [comment]
audit log:
>* GitHub fork (not canonical repository)

base repository not works with >=php7

File: Formula/php70-xhprof.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+require File.expand_path("../../Abstract/abstract-php-extension", __FILE__)
+
+class Php70Xhprof < AbstractPhp70Extension
+  init
+  desc "XHProf is a function-level hierarchical profiler for PHP and has a simple HTML based user interface."
+  homepage "https://github.com/longxinH/xhprof"
+  url "https://github.com/longxinH/xhprof/archive/87735837d37ebfc82cb7a02dc28bbf66f74540ef.tar.gz"
```
Comment (SMillerDev) [inline Formula/php70-xhprof.rb]: Can you please ask for a versioned release? This way you're probably better off with a head-only formula

File: Formula/php70-xhprof.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  sha256 "6f2fbf9e4bf4199208b7cd5c0f6ce5677290488f255e4eb89953866177fcf812"
+  head "https://github.com/longxinH/xhprof.git"
+  version "8773583"
```
Comment (SMillerDev) [inline Formula/php70-xhprof.rb]: I see the last official release there is 1.1, I think it'd make everything a lot easier in maintenance if we name this version something like 1.1.900, expecting that won't really clash.

Or better yet, have them version php72 support as 1.2

File: Formula/php70-xhprof.rb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+  bottle do
+    cellar :any_skip_relocation
```
Comment (SMillerDev) [inline Formula/php70-xhprof.rb]: Can you check if it works without this?

File: Formula/php70-xhprof.rb [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    rebuild 1
```
Comment (SMillerDev) [inline Formula/php70-xhprof.rb]: This isn't needed

**Comment by KonstantinKuklin** [comment]
@SMillerDev what do u mean in "add back the checklist in main post"?
Also it's not clear for me: are u propose to remove url\version and use only head, or give version name like 1.1.XX?

File: Formula/php70-xhprof.rb (hunk 3 above)
Comment (javian) [reply Formula/php70-xhprof.rb]: The whole bottle block should be removed in new formulas so we shouldn't need to mess with individual configurations and the bot will do whatever is necessary to include the right links to the binaries.

**Comment by KonstantinKuklin** [comment]
I requested a new tag for head, but I can't give any guarantees and time estimates.

**Comment by KonstantinKuklin** [comment]
@SMillerDev @javian any updates?

**Comment by KonstantinKuklin** [comment]
nice

**Comment by MikeMcQuaid** [comment]
As [announced on the 19th of January](https://brew.sh/2018/01/19/homebrew-1.5.0/) this tap will be archived by 31st March 2018. Various PHP formulae have been [migrated to Homebrew/homebrew-core](https://github.com/Homebrew/homebrew-php/blob/master/tap_migrations.json). Thanks for your pull request anyway and sorry we couldn't include it. Please consider [maintaining this formula in your own tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap).

**Pull request closed without merging**